### DISCUSSION TITLE
PageRank initial weight

### DISCUSSION THREAD

**Discussion by Eloh666** [networkit/networkit#690] [Q&A]
Hi there,

I am currently looking to use NetworkIt to calculate PageRank for a fairly sizeable graph represents a website domain.
The website comprises of roughly ~700k nodes and a 10mil edges.

For the sake of simulation, I am looking to see if there is a way to set an initial larger weight on some of the nodes. This is especially necessary to artificially boost the homepage weight to simulate its relevance in the larger super-graph representing the internet.

Is there any way to raise the value of the homepage weight to achieve the result?

**Comment by avdgrinten** [comment]
Are you looking for personalized PageRank? Just changing the initial values will not affect the solution once the algorithm converges.

The size of your graph should not pose a challenge to NetworKit.
